qprofiler2/src/org/qcmg/qprofiler2/summarise/ReadGroupSummary.java

```diff
@@ -197,21 +197,19 @@ public void preSummary() {
 				tLenOverall.increment(i, p.getTLENCounts().get(i));					 
 			}			
 		}
-
 		this.overlapStats = new SummaryReportUtils.TallyStats(overlapBase);
 		this.pairtLenStats = new SummaryReportUtils.TallyStats(tLenOverall );
 		this.softclipStats = new SummaryReportUtils.TallyStats( softClip);
 		this.hardclipStats = new SummaryReportUtils.TallyStats( hardClip);
 		this.readlengthStats = new SummaryReportUtils.TallyStats( readLength );	
 		
 		int maxLenght = (int)readlengthStats.getMax();
-		QCMGAtomicLongArray trimedBase = new QCMGAtomicLongArray(maxLenght);	
-		for (int i = 1 ; i < forTrimLength.length() ; i++)	{			
+		QCMGAtomicLongArray trimedBase = new QCMGAtomicLongArray(maxLenght+1);	
+		for (int i = 0 ; i < forTrimLength.length() ; i++)	{			
```

Comment:
Is this change covered by a test?

Reply:
unMappedReadTest()
 noSeqReadTest()
cover above case